bulk-ingester/src/main/resources/application.yml

```diff
@@ -1,23 +1,30 @@
 ### db-connector config
-spring.datasource:
-  url: jdbc:postgresql://localhost:5432/alfresco
-  username: alfresco
-  password: alfresco
-  hikari.maximumPoolSize: 20
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/alfresco
+    username: alfresco
+    password: alfresco
+    hikari.maximumPoolSize: 20
 
-alfresco.reindex.pathCacheSize: 10000
+alfresco:
+  reindex:
+    pathCacheSize: 10000
```

Comment:
The original single line property looked ok to me. Do you prefer presenting it across multiple lines?

Reply:
Ah - I see that the `alfresco` part was duplicated. So we could keep some of it compressed:
```
alfresco:
  reindex.pathCacheSize: 10000
  bulk.ingest:
    node-params:
      fromId: 0
      toId: 20000000000
  ...
```

Reply:
One thing - it was duplicated. Second one - I think we shouldn't mix text-style properties with yaml-style properties.

Reply:
Ok - there's also `hikari.maximumPoolSize` above.